pyphare/pyphare/pharesee/run/run.py

```
@@ -372,55 +312,80 @@ def GetDl(self, level="finest", time=None):
-            time_keys = ff["t"].keys()
-            time = np.zeros(len(time_keys))
-            for it, t in enumerate(time_keys):
-                time[it] = float(t)
-            ts[quantities_per_file[basename]] = time
-            ff.close()
-        return ts
+            # assumes all qties in the hierarchy have the same ghost width
+            # so take the first patch data of the first patch of the first level....
+            nbrGhosts = list(hierarchy.level(0).patches[0].patch_datas.values())[
+                0
+            ].ghosts_nbr
```

Ruff (0.16.0)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)